homeassistant/components/calendar/__init__.py

```diff
@@ -69,6 +74,18 @@ def __init__(self, hass, data):
             'description': '',
         }
 
+        class CalendarEventView(http.HomeAssistantView):
```

Comment:
Please don't define inside the constructor but outside of it. You can pass the device via the constructor.